### ISSUE TITLE
Interface connectors - drop from 2 to 1

### ISSUE THREAD

**Issue by koniarik** [TVavrinec/Servio-PCB#11]
Currently we have two interface connectors (4x1, 1.27mm, JST PH), after some more thought with this:
- It allows chaining of servos, which is _nice_ but not always practical or necessary. one especially has to keep in mind that in case multiple servos are chained, the first one has _all the current_ flowing throu it. Overall nice to have feature but not necessary
- Even if we use 1.27mm connectors, they indeed take a lot of place.

I am thinking that maybe we should use only once connector.

Thing is, I was insisting on this also as a convenience thing for development, as for a lot of testjigs, I use one connector only for power, and one only for comms. (It is quite practical) Thing is, for that I don't need two connectors on the board itself, but two connectors on the _debug board_ surrounding it - this can be avoided entirely.

**Comment by koniarik** [comment]
Just done, one is indeed good enough
